### ISSUE TITLE
Build error with Xcode 10 beta 5

### ISSUE THREAD

**Issue by keith** [Instabug/Instabug-iOS#280]
### Steps to Reproduce the Problem

Build an app with InstabugStatic.framework for multiple architectures in Xcode 10 beta 5 (there are some other conditions to this that I haven't been able to figure out yet).

```
duplicate symbol l205 in:
    .../InstabugStatic/current/InstabugStatic.framework/InstabugStatic(PLCrashReport.o)
    .../InstabugStatic/current/InstabugStatic.framework/InstabugStatic(PLCrashReportTextFormatter.o)
duplicate symbol l206 in:
    .../InstabugStatic/current/InstabugStatic.framework/InstabugStatic(PLCrashReport.o)
    .../InstabugStatic/current/InstabugStatic.framework/InstabugStatic(PLCrashReportTextFormatter.o)
duplicate symbol l207 in:
    .../InstabugStatic/current/InstabugStatic.framework/InstabugStatic(PLCrashReport.o)
    .../InstabugStatic/current/InstabugStatic.framework/InstabugStatic(PLCrashReportTextFormatter.o)
duplicate symbol l208 in:
    .../InstabugStatic/current/InstabugStatic.framework/InstabugStatic(PLCrashReport.o)
    .../InstabugStatic/current/InstabugStatic.framework/InstabugStatic(PLCrashReportTextFormatter.o)
ld: 4 duplicate symbols for architecture arm64
```

Related issue with crashlytics https://stackoverflow.com/questions/51602785/xcode-10b5-duplicate-symbol-linker-error-cant-compile-with-crashlytics https://openradar.appspot.com/42768051

Because of a change in the Xcode 10 beta 5 linker, Objective-C libraries built with old versions of the compiler (I'm not sure what the cutoff is) cause this duplicate symbol warning. From what I've header this is intentional behavior, so I assume in this case the PLCrashReporter library bundled with Instabug was built with an older version of Xcode.

### SDK Version

8.0.7

### iOS Version

iOS 12

**Comment by Kmohamed** [comment]
I think you are right PLCrashReporter build with old version of Xcode let me try it

**Comment by Kmohamed** [comment]
@keith I already build PLCrashReporter with a new version of Xcode and works fine on Xcode 10 beta 5
but I checked Instabug version 8.0.7 and also works fine on Xcode 10 beta 5. do you make any custom configuration ? I wanna reproduce it in order to be sure that I fix the issue.

**Comment by keith** [comment]
I couldn't reproduce with a bit of work in a sample project, if you want to upload a binary that should be fixed here I can test it

**Comment by Kmohamed** [comment]
Sure here is the [binary](https://www.dropbox.com/s/b0qrddt29mwgmvg/InstabugStatic.framework.zip?dl=0)

**Comment by keith** [comment]
Confirmed that it is fixed with this binary

**Comment by Kmohamed** [comment]
Fix released in SDK version 8.0.8

**Comment by keith** [comment]
Thanks!
